lib/player.js

```diff
@@ -5651,15 +5651,20 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
    * Select a specific text track. <code>track</code> should come from a call to
    * <code>getTextTracks</code>. If the track is not found, this will be a
    * no-op. If the player has not loaded content, this will be a no-op.
+   * If you no longer want to select a text track, provide `null`.
    *
    * <p>
    * Note that <code>AdaptationEvents</code> are not fired for manual track
    * selections.
    *
-   * @param {shaka.extern.TextTrack} track
+   * @param {?shaka.extern.TextTrack} track
    * @export
    */
   selectTextTrack(track) {
+    if (track == null) {
+      this.setTextTrackVisibility(false);
+      return;
+    }
```

Comment:
We don't fire `textchanged` event when text track is null, I think we should now to remain consistent.